sopel/bot.py

```diff
@@ -731,6 +731,9 @@ def search_url_callbacks(self, url):
         .. __: https://docs.python.org/3.6/library/re.html#match-objects
 
         """
+        if 'url_callbacks' not in self.memory:
+            # nothing to search
+            return
```

Comment:
This just makes me happier for some reason.

```suggestion
            return

```